07-slotfills-esnext/index.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Plugin Name: Gutenberg Examples SlotFills ESNext
+ * Plugin URI: https://github.com/WordPress/gutenberg-examples
+ * Description: This is a plugin demonstrating how to use SlotFills in the Gutenberg editor.
+ * Version: 1.0.0
+ * Author: the Gutenberg Team
+ *
+ * @package gutenberg-examples
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Registers all block assets so that they can be enqueued through Gutenberg in
+ * the corresponding context.
+ */
+function gutenberg_examples_07_esnext_register_block() {
+
+	if ( ! function_exists( 'register_block_type' ) ) {
+		// Gutenberg is not active.
+		return;
+	}
+
+	// automatically load dependencies and version.
+	$asset_file = include(plugin_dir_path(__FILE__) . 'build/index.asset.php');
```

Comment:
Minor coding standards issue
```suggestion
	$asset_file = include( plugin_dir_path( __FILE__ )  .  'build/index.asset.php' );
```